### PR TITLE
MWPW-109312 - Section Metadata

### PR DESCRIPTION
**Pull request by auniverseaway**
* Ability to spec section style
* Ability to set background image

Resolves: MWPW-109312

Test URLs:
- Before: https://main--milo--adobecom.hlx.page/
- After: https://section-meta--milo--adobecom.hlx.page/drafts/cmillar/section-container

### CODE REVIEW

File: libs/blocks/marquee/marquee.css [hunk 1]
```diff
@@ -148,6 +148,7 @@
 .marquee .image img {
   display: block;
   width: 100%;
+  min-height: 150px; /* fixed with breaking 05 */
```
Comment (chrischrischris) [inline libs/blocks/marquee/marquee.css:151]: breaking 05?  What's that referring to?

File: libs/blocks/section-metadata/section-metadata.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    if (color && section) {
+      section.style.backgroundColor = color;
+    }
+  }
+}
+
+function handleStyle(div) {
+  const value = div.textContent.toLowerCase();
+  const split = value.split(', ');
+  const styles = split.map((style) => style.replaceAll(' ', '-'));
```
Comment (chrischrischris) [inline libs/blocks/section-metadata/section-metadata.js:21]:
```suggestion
  const styles = value.split(', ').map((style) => style.replaceAll(' ', '-'));
```
nit: may as well combine these lines

File: libs/blocks/section-metadata/section-metadata.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      section.classList.add('has-background');
+      pic.classList.add('section-background');
+      section.insertAdjacentElement('afterbegin', pic);
+    }
+  } else {
+    const color = div.textContent;
+    if (color && section) {
+      section.style.backgroundColor = color;
+    }
+  }
```
Comment (chrischrischris) [inline libs/blocks/section-metadata/section-metadata.js:15]: nit: you could early return and avoid having to check for section twice:
```suggestion
  const section = div.closest('.section');
  if (!section) return;
  if (pic) {
    section.classList.add('has-background');
    pic.classList.add('section-background');
    section.insertAdjacentElement('afterbegin', pic);
  } else {
    const color = div.textContent;
    if (color) {
      section.style.backgroundColor = color;
    }
  }
```

File: libs/blocks/marquee/marquee.css (hunk 1 above)
Comment (auniverseaway) [reply libs/blocks/marquee/marquee.css]: There's a breaking change that's going to be happening in Helix that will obviate the need to set this height.

**Review by chrischrischris — APPROVED** [review]

**Review by amauch-adobe — APPROVED** [review]
Approved code review.  QA approved and validated, tested on chrome, firefox, edge, and safari using test url.

**Pull request merged**